ratis-server/src/main/java/org/apache/ratis/server/impl/LeaderStateImpl.java

```diff
@@ -247,6 +247,8 @@ boolean removeAll(Collection<LogAppender> c) {
   private final EventProcessor processor;
   private final PendingRequests pendingRequests;
   private final WatchRequests watchRequests;
+  private final ReadRequests readRequests;
+  private final Map<Long, CompletableFuture<Boolean>> pendingBroadcasts;
```

Comment:
Move `LeaderStateImpl.ReadRequests` and `ReadRequests.pendingBroadcasts` should be combined into a single data structure.

Reply:
done